### DISCUSSION TITLE
Tips for Networking (Social Media)

### DISCUSSION THREAD

**Discussion by AllanRegush** [EddieHubCommunity/support#131] [General]
I was wondering if anyone had any tips for networking on social media?

**Comment by eddiejaoude** [comment]
Great discussion topic @AllanRegush 👍 

I am still learning and improving on this area, but my initial thoughts:
- get involved io the conversation, for example replying to people 
- post valuable content, for example tips 
- post engaging content that asks questions (I dont do this, but I probably should)

I will be interested to hear what other people think / do / try etc
